apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva } from 'class-variance-authority';
+
+import { ReactComponent as DoneIcon } from '~/assets/SVGIcons/CheckFill.svg';
```

Comment:
UI components generally have been moving icons to be co-located in `icons/`, as the assets in `/assets` are a little inconsistent in application.